RELEASE.md

```diff
@@ -0,0 +1,21 @@
+# Release process for Rust SDK
+
+## Setup
+1. Login to your *crates.io* account: `cargo login`
+>- Prior authorization must be given by a maintainer of the crate
+
+## Release
+1. Create a new git branch: `release/vX.Y.Z`.
+2. Run all tests against [hedera-local-node](https://github.com/hashgraph/hedera-local-node). Stop local-node once the tests are completed.
+>- `cargo test`
+3. Change the version edition in *Cargo.toml*.
+>- `version = x.xx.x`
+>- For example, version `0.22.0` changes to `0.23.0` upon each release (at the moment).
```

Comment:
Probably sufficient to say that we follow [semver 2](https://semver.org/spec/v2.0.0.html)